app/services/google_services.rb

```diff
@@ -19,7 +19,7 @@ def store_details(place_id)
   end
 
   def convert_zipcode_to_coordinates(zipcode)
-    zipcode_info = Faraday.get("https://www.zipcodeapi.com/rest/#{ENV['zipcode_key']}/info.json/#{zipcode}/degrees")
+    zipcode_info = Faraday.get("http://api.zippopotam.us/us/#{zipcode}")
```

Comment:
That was an awesome find on your part. That other api wasnt very good and low request limit.